develop/develop-images/dockerfile_best-practices.md

```diff
@@ -418,7 +418,7 @@ The following examples show the different acceptable formats. Explanatory commen
 > Strings with spaces must be quoted **or** the spaces must be escaped. Inner
 > quote characters (`"`), must also be escaped.
 
-```Dockerfile
```

Comment:
looks like rouge is case-sensitive; we should look if newer versions are more flexible (GitHub itself is not case-sensitive, so perhaps that's the case)